kong/templates/nginx_kong.lua

```diff
@@ -84,7 +84,7 @@ server {
     access_log logs/access.log;
 
 > if ssl then
-    listen ${{PROXY_LISTEN_SSL}} ssl;
+    listen ${{PROXY_LISTEN_SSL}} ssl http2;
```

Comment:
Yes, I understand. It may be desirable that Kong operators do not want to expose clients to potential instabilities associated with the h2 implementation in Nginx, documented or otherwise.